tests/unit_tests/test_load_forward_model.py

```diff
@@ -170,7 +170,7 @@ def test_load_forward_model_summary(summary_configuration, storage, expected, ca
         responses=ert_config.ensemble_config.response_configuration
     )
     prior_ensemble = storage.create_ensemble(
-        experiment_id, name="prior", ensemble_size=100
+        experiment_id, name="prior", ensemble_size=1
```

Comment:
Why the change in size?

Reply:
It was related to the `getEnsembleSize()` change, undid both